.github/workflows/build-and-test.yml

```diff
@@ -112,6 +112,54 @@ jobs:
         with:
           files: ./service/build/reports/jacoco/test/html/index.html
 
+  source-clear:
+    needs: [ build ]
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK
+        uses: actions/setup-java@v2
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+
+      - name: Gradle cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: v1-${{ runner.os }}-gradle-${{ github.ref }}-${{ github.sha }}
+
+      - name: SourceClear scan
+        env:
+          SRCCLR_API_TOKEN: ${{ secrets.SRCCLR_API_TOKEN }}
+        run: ./gradlew --build-cache srcclr
+
+  sonar-scanner:
+    needs: [ build ]
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK
+        uses: actions/setup-java@v2
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+
+      - name: Gradle cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: v1-${{ runner.os }}-gradle-${{ github.ref }}-${{ github.sha }}
+
+      - name: SonarQube scan
+        run: ./gradlew -Dsonar.login="${{ secrets.SONAR_TOKEN }}" --build-cache sonarqube
```

Comment:
I'm getting confused about the global vs. service build.gradle files. If the sonarqube task is defined in the service, why isn't this :service:sonarqube? 🤔

Reply:
Either would work in this case. The task is only run in the subprojects that have it present, so using a global task means that if we add this scan to other projects, we don't have to update the Gradle command. This seems like a nice feature to me, but I could make it more specific instead if you think it's confusing.

Reply:
Thanks for clarifying! I think we can leave it as-is so we don't have to update the command later